root/static/scripts/relationship-editor/components/RelationshipItem.js

```diff
@@ -272,6 +277,25 @@ const RelationshipItem = (React.memo<PropsT>(({
               })
             )
             : targetDisplay}
+          {hasPendingEdits && nonEmpty(openEditsLink) ? (
+            <>
+              {' '}
+              <Tooltip
+                content={exp.l(
+                  'This relationship has {edit_search|pending edits}.',
+                  {edit_search: openEditsLink},
+                )}
+                target={
+                  <img
```

Comment:
The icon looks a lot better aligned for me with `vertical-align: middle` (in both Firefox and Chromium).